self-core-impl/src/main/java/com/selfxdsd/core/BitbucketOrganizationRepos.java

```diff
@@ -0,0 +1,73 @@
+package com.selfxdsd.core;
+
+import com.selfxdsd.api.Repo;
+import com.selfxdsd.api.Repos;
+import com.selfxdsd.api.User;
+import com.selfxdsd.api.storage.Storage;
+
+import java.net.URI;
+import java.util.Iterator;
+import java.util.Spliterator;
+import java.util.function.Consumer;
+
+/**
+ * A Bitbucket Provider Organization Repos.
+ *
+ * @author Nikita Monokov (nmonokov@gmail.com)
+ * @version $Id$
+ * @since 0.0.62
+ */
+public class BitbucketOrganizationRepos implements Repos {
```

Comment:
@nmonokov Can you write a TODO here, to continue implementing this class?

Reply:
I left it with unsupported exceptions. I couldn't understand where is the line between my task and repos and just made this class with defaults :)